geomstats/geometry/riemannian_metric.py

```diff
@@ -161,18 +111,22 @@ def christoffels(self, base_point):
         christoffels: array-like, shape=[..., dim, dim, dim]
             Christoffel symbols.
         """
-        cometric_mat_at_point = self.inner_product_inverse_matrix(base_point)
+        cometric_mat_at_point = self.inner_product_inverse_matrix(
```

Comment:
Maybe `inner_product_inverse_matrix` should be renamed to `metric_inverse_matrix` for consistency and brevity?

Reply:
Definitely!